### ISSUE TITLE
Conversion can overflow rule doesnt work for enums in sonarlint

### ISSUE THREAD

**Issue by movedoa** [Riverside-Software/sonar-openedge#1095]
V 2.24

On the server analysis i get a lot of `Use enumRef:getValue() instead of INTEGER(enumRef)`.
In SonarLint these errors are not reported, no error message in the output.
STRING(cLongCharVar) for example works in SonarLint.

```
SonarLint processing file /Foo/SomeClass.cls...
Found 0 issue(s)
```

![grafik](https://github.com/Riverside-Software/sonar-openedge/assets/13309951/07c6506f-2915-483d-b072-8126f9133909)

**Comment by gquerret** [comment]
Fixed in [SonarLint 9.1.2906](https://github.com/Riverside-Software/sonar-openedge/releases/tag/V2.24.1)
